test/multiarray/test_chunked.cxx

```diff
@@ -763,10 +764,10 @@ class ChunkedMultiArrayTest
     
         int go = 0;
         
-        threading::thread t1(testMultiThreadedRun, a.get(), 0, 4, &go);
-        threading::thread t2(testMultiThreadedRun, a.get(), 1, 4, &go);
-        threading::thread t3(testMultiThreadedRun, a.get(), 2, 4, &go);
-        threading::thread t4(testMultiThreadedRun, a.get(), 3, 4, &go);
+        threading::thread t1(std::bind(testMultiThreadedRun,a.get(),0,4,&go));
```

Comment:
@ukoethe I had to change slightly this as apparently boost thread does not support 5 arguments in the constructor.